server.js

```diff
@@ -26,6 +16,12 @@ app.use(require("webpack-dev-middleware")(compiler, {
 }));
 app.use(require("webpack-hot-middleware")(compiler));
 
+//  Create hot-reloading bundle for server rendering
+var basePath = require('path').resolve(__dirname, '.');
+var webpackIsomorphicTools = new WebpackIsomorphicTools(require('./webpack-isomorphic-tools'))
+    .development(true)
+    .server(basePath, function() { });
```

Comment:
It's also not clear to me what's actually in this bundle, can you expand the comment please?